torch/csrc/jit/passes/quantization/helper.h

```diff
@@ -99,6 +99,11 @@ TORCH_API bool useQuantizable(const Use& use, QuantType quant_type);
 // Given a CallFunction node, extract the graph of the called function
 TORCH_API std::shared_ptr<Graph> getCallFunctionGraph(Node* n);
 
+TORCH_API bool matchCallFuncToUse(
```

Comment:
optional nit: docblock?

Reply:
nit: I think we can remove TORCH_API actually, it's a mistake in the beginning to add TORCH_API for all these helper functions..